js/pages/configuration/roles/role-details.js

```diff
@@ -324,14 +324,16 @@ define([
         }
 
         async delete() {
-            this.loading(true);
-            await roleService.delete(this.roleId());
-            var roles = this.model.roles().filter((role) => {
-                return role.id != this.roleId();
-            });
-            this.model.roles(roles);
-            this.close();
-            this.loading(false);
+            if (confirm("Are you sure to delete role?")) {
```

Comment:
Move to util method, same as here
https://github.com/OHDSI/ArachneUI/blob/d84c956e95a8ca8cbb4cb436b9927aed329299f1/src/modules/Admin/components/AdminList/Table/container.js#L66